build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in

```diff
@@ -1,13 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2018 Werner Schweer and others -->
-<application>
-  <id type="desktop">mscore.desktop</id>
-  <metadata_license>CC0-1.0</metadata_license>
+<component type="desktop-application">
+  <id>org.musescore.MuseScore</id>
+  <translation type="qt">mscore</translation>
```

Comment:
ID should be `org.musescore.MuseScore${MSCORE_INSTALL_SUFFIX}`, and translation should be `mscore${MSCORE_INSTALL_SUFFIX}` assuming it refers to the name of the executable on the command line.

Reply:
Oh, ok, if the suffix is in the ID too, it should be fine to rename the file as well.

The translation tag refers to to the prefix of the `.qm` translation files.